src/controllers/fxaRpEvents.js

```diff
@@ -96,21 +96,21 @@ const fxaRpEvents = async (req, res) => {
   } catch (e) {
     log.error('fxaRpEvents', e)
     captureException(e)
-    res.status(202)
+    res.status(400).send('Bad Request')
   }
 
   if (!decodedJWT?.events) {
     // capture an exception in Sentry only. Throwing error will trigger FXA retry
     log.error('fxaRpEvents', decodedJWT)
     captureMessage('fxaRpEvents: decodedJWT is missing attribute "events"', decodedJWT)
-    res.status(202)
+    res.status(400).send('Bad Request')
```

Comment:
Hm, would it be better to throw `UserInputError`?

Reply:
Fixed. thanks!